scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingMethodImpl.java

```diff
@@ -692,6 +693,17 @@ protected boolean createExecution(NodeSet nodeSet, Node node, InternalJob job, I
 
     }
 
+    private void waitForTaskToBeStarted(Future<Void> taskExecutionSubmittedFuture) {
+        try {
+            // before signaling that the task is started, we need
+            // to make sure the task is correctly submitted to the
+            // task launcher
+            taskExecutionSubmittedFuture.get(DOTASK_ACTION_TIMEOUT, TimeUnit.MILLISECONDS);
+        } catch (Exception e) {
+            logger.warn("Could not wait for the task to be started on TaskLauncher.", e);
```

Comment:
I prefer "Error while waiting for the task to be started.", the exception should inform if it's a timeout, etc.

Reply:
ok, I'll change that into multiple catch then.